buildkite/scripts/merges-cleanly.sh

```diff
@@ -4,21 +4,23 @@ BRANCH=$1
 CURRENT=$(git branch --show-current)
 echo 'Testing for conflicts between the current branch `'"${CURRENT}"'` and `'"${BRANCH}"'`...'
 
+
 # Adapted from this stackoverflow answer: https://stackoverflow.com/a/10856937
 # The git merge-tree command shows the content of a 3-way merge without
 # touching the index, which we can then search for conflict markers.
 
 # Tell git where to find ssl certs
 git config --global http.sslCAInfo /etc/ssl/certs/ca-bundle.crt
-# Fetch a fresh copy of the repo
-git fetch origin
```

Comment:
Why not fetching fork? @dkijania

Reply:
Also in line #31 below: The stderr produced might be wrong as it does not consider $REMOTE

Reply:
In this job we want to compare branch which is already check out against mina repo so we don't need to refresh fork

Reply:
Stderr is correct as we compare forked branch against mina branch